ee_plugin/utils.py

```diff
@@ -277,18 +279,28 @@ def ee_image_to_geotiff(
 
     tiles = tile_extent(ee_image, extent, scale)
     logger.info(f"Generated {len(tiles)} tiles for export.")
+    # TODO: review threshold and messaging
```

Comment:
Is this something we can address before merge?

Reply:
tracking conversation here, i'll just remove the TODO for now as 5 seems like an early enough warning.